packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -159,6 +159,14 @@ export interface RestApiOptions extends ResourceOptions {
    * @default - when no export name is given, output will be created without export
    */
   readonly endpointExportName?: string;
+
+  /**
+   * The EndpointConfiguration property type specifies the endpoint types of a REST API
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-apigateway-restapi-endpointconfiguration.html
+   *
+   * @default - No endpoint configuration
```

Comment:
This statement conflicts with the one in the README that says that the default is an edge optimized endpoint.

Reply:
I added more clarity to the message. It now reads like this

`No endpoint types that translates to a EndpointType.EDGE endpoint type.`